test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -25,6 +27,8 @@ describe.skip('Create token, approve token and approve token without gas', funct
           dapp: true,
           fixtures: 'connected-state',
           ganacheOptions,
+          smartContract,
+          failOnConsoleError: false,
```

Comment:
I am not 100% sure we need the `failOnConsoleError` on all the tests, as it looks like they do not throw any console errors naturally (at least in my local env), so in the case there is a console error might be that some "real" error is happening?

Reply:
We had failure in one test and I put failOnConsoleError just in that test. After I thought maybe it would be good to put it in all tests, but if that is not usual i can remove it.

Reply:
I verified and it's not failing for me without the `failOnConsoleError`. If they are not failing for you I guess we can remove it